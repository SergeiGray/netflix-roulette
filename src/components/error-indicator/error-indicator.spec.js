import React from 'react';
import ErrorIndicator from './error-indicator.js';

describe('ErrorIndicator component', () => {
  it('should render ErrorIndicator component', () => {
    const component = shallow(<ErrorIndicator/>);

    expect(component).toMatchSnapshot();
  });
});