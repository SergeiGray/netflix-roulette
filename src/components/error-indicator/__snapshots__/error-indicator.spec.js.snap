// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorIndicator component should render ErrorIndicator component 1`] = `
<div
  className="error-indicator text-danger"
>
  <img
    alt="Error"
    src="test-file-stub"
  />
  <span>
    An error occurred.
  </span>
  <span>
    The support team is already working on the fix.
  </span>
</div>
`;
